docs/releases/1.9.txt

```diff
@@ -161,6 +161,23 @@ Minor features
   changing the ``select_related()`` values used in the admin's changelist query
   based on the request.
 
+* Added ``view`` permission to the
```

Comment:
This documentation is looking better, but the bulk of it should be moved into the admin docs so that users of future versions of Django don't have to refer back to the release notes to see how it works. We need to target 1.10 now that 1.9 is feature frozen.